src/libs/antares/study/output.cpp

```diff
@@ -89,15 +89,15 @@ class OutputFolderIterator : public IO::Directory::IIterator<true>
 } // anonymous namespace
 
 Output::Output(const AnyString& folder) :
- version(0), timestamp(0), mode(Data::SimulationMode::Economy), menuID(-1), viewMenuID(-1), outputViewerID(-1)
+ timestamp(0), mode(Data::SimulationMode::Economy), menuID(-1), viewMenuID(-1), outputViewerID(-1)
 {
     loadFromFolder(folder);
 }
 
 bool Output::valid() const
 {
     // The outputs as we know them was first introduced in Antares 3.0
-    return (uint)version <= (uint)Data::versionLatest;
+    return version <= Data::versionLatest;
```

Comment:
```suggestion
    return version <= StudyVersion::latest();
```
(static data member ?)